schemas/groups/steering.json

```diff
@@ -97,12 +90,6 @@
                 "gps"
               ]
             },
-
-            "source": {
-              "description": "Source of this data",
-              "$ref": "../definitions.json#/definitions/source"
-            },
-
```

Comment:
Oh that was a quick fix/destruct to get the test to pass. Not sure why it doesn't pass with these in place... Help needed